CONTRIBUTING.md

```diff
@@ -207,20 +210,50 @@ easy to reference the pull request where a change was introduced.
      release, and add the github aliases of external contributors to the top of
      the changelog. To lookup an alias from an email, try `bash ./scripts/authors.sh <email>`
    - reset the `CHANGELOG_PENDING.md`
-   - bump versions
+   - bump the appropriate versions in `version.go`
 4. push your changes with prepared release details to `vX.X` (this will trigger the release `vX.X.0`)
 5. merge back to master (don't squash merge!)
 
 #### Minor Release
 
-If there were no breaking changes and you need to create a release nonetheless,
-the procedure is almost exactly like with a new release above.
+Minor releases are done differently from major releases. Minor release pull requests should be labeled with `R:minor` if they are to be included.
+
+1. Checkout the last major release, `vX.X`.
+
+   - `git checkout vX.X`
+
+2. Create a branch off the latest release, `rc1/vX.X.x`, and push the branch.
```

Comment:
Why `rc1`?

Reply:
why we need an rc branch or the name?

Reply:
Nevermind. I took this to be that all minor releases should have a `rc1/` prefix.